include/cantera/thermo/PengRobinson.h

```diff
@@ -0,0 +1,359 @@
+     *  mechanical equation of state \f$ P(T, \rho, Y_1, \dots, Y_K) \f$.
+     *
+     * \f[
+     *    P = \frac{RT}{v-b_{mix}} - \frac{\left(\alpha a\right)_{mix}}{v^2 + 2b_{mix}v - b_{mix}^2 }
+     * \f]
+     *
+     *  where:
+     *
+     * \f[
+     *    \alpha = \left[ 1 + \kappa \left(1-T_r^{0.5}\right)\right]^2
+     * \f]
+     *
+     *  and
+     *
+     * \f[
+     *    \kappa = \left(0.37464 + 1.54226\omega - 0.26992\omega^2\right)                              for omega <= 0.491
+     *    \kappa = \left(0.379642 + 1.487503\omega - 0.164423\omega^2 +  0.016667\omega^3 \right)      for omega > 0.491
+     * \f]
+     *
+     *Coefficients a_mix, b_mix and (a \alpha)_{mix} are caclulated as
+     *
+     *\f[
+     *a_{mix} = \sum_i \sum_j X_i X_j a_{i, j} = \sum_i \sum_j X_i X_j sqrt{a_i a_j}
+     *\f]
+     *
+     *\f[
+     *   b_{mix} = \sum_i X_i b_i
+     *\f]
+     *
+     *\f[
+     *   {a \alpha}_{mix} = \sum_i \sum_j X_i X_j {a \alpha}_{i, j} = \sum_i \sum_j X_i X_j sqrt{a_i a_j} sqrt{\alpha_i \alpha_j}
+     *\f]
+     *
+     *
+     */
+    
+    virtual double pressure() const;
+
+    // @}
+
+public:
+
+    //! Returns the standard concentration \f$ C^0_k \f$, which is used to
+    //! normalize the generalized concentration.
+    /*!
+     * This is defined as the concentration by which the generalized
+     * concentration is normalized to produce the activity. In many cases, this
+     * quantity will be the same for all species in a phase. 
+     * The ideal gas mixture is considered as the standard or reference state here. 
+     * Since the activity for an ideal gas mixture is simply the mole fraction, for an ideal gas
+     * \f$ C^0_k = P/\hat R T \f$.
+     *
+     * @param k Optional parameter indicating the species. The default is to
+     *          assume this refers to species 0.
+     * @return
+     *   Returns the standard Concentration in units of m3 kmol-1.
+     */
+    virtual double standardConcentration(size_t k=0) const;
+
+    //! Get the array of non-dimensional activity coefficients at the current
+    //! solution temperature, pressure, and solution concentration.
+    /*!
+     * For all objects with the Mixture Fugacity approximation, we define the
+     * standard state as an ideal gas at the current temperature and pressure of
+     * the solution. The activities are based on this standard state.
+     *
+     * @param ac Output vector of activity coefficients. Length: m_kk.
+     */
+    virtual void getActivityCoefficients(double* ac) const;
+
+    /// @name  Partial Molar Properties of the Solution
+    //@{
+
+    //! Get the array of non-dimensional species chemical potentials.
+    //! These are partial molar Gibbs free energies.
+    /*!
+     * \f$ \mu_k / \hat R T \f$.
+     * Units: unitless
+     *
+     * We close the loop on this function here calling getChemPotentials() and
+     * then dividing by RT. No need for child classes to handle.

```

Comment:
The comment here about child classes - do we expect people to subclass `PengRobinson`? If not, can that be removed or moved somewhere else?

Reply:
You are right. There won't be any subclass for `PengRobinson`, so I removed these comments.